### ISSUE TITLE
[algorithms] Inconsistency for parameter `x` vs parameter named `x`

### ISSUE THREAD

**Issue by johelegp** [cplusplus/draft#4086]
It seems that the former is prevalent.

**Comment by jensmaurer** [comment]
Editorial meeting 2023-11-06: We want to say "no parameter named X", to highlight that the name isn't really relevant for C++; naming the parameter is just a method of specification.

Pull request welcome.

**Comment by JohelEGP** [comment]
I have reviewed the use of "parameter" in [algorithms] to remind myself what this issue is about.
It seems that the editorial meeting consensus is with regards to template parameter names such as `BinaryPredicate`.
I think those are perfectly covered by [algorithms.requirements].
My target are function parameter names.
Sorry for not being clear enough.
Here is an example of the inconsistency.
https://github.com/cplusplus/draft/blob/dc6eed02986d9c3c6827c710adb577ba0809f939/source/algorithms.tex#L4288-L4291

**Comment by jensmaurer** [comment]
No, we did discuss exactly the function parameter naming issue that your example is about.
